.env.sample

```diff
@@ -39,3 +39,10 @@ APIM_INFORMATICA_USERNAME=
 APIM_INFORMATICA_PASSWORD=
 APIM_INFORMATICA_MAX_REDIRECTS=
 APIM_INFORMATICA_TIMEOUT=
+
+# ORDNANCE SURVEY
+# Use .uk domain, instead of .co.uk
```

Comment:
Why is this?

Reply:
.co.uk redirects to .uk and some tests are not happy with redirect response. We could change tests, but maybe avoiding redirect is good solution?

Reply:
I agree that this is the best solution

Reply:
OS domain `https://api.os.uk` is public so I added it to .env.sample and added some instructions to README.md.

Not sure if README.md part is required, the bigger this doc is, less people might read it.